### ISSUE TITLE
The arrow on the popout has incorrect aria role [ Accessibility ]

### ISSUE THREAD

**Issue by mavtron-andrewrusso** [tprouvot/Salesforce-Inspector-reloaded#454]
- [X] I've read the [common issues](https://tprouvot.github.io/Salesforce-Inspector-reloaded/troubleshooting/) and the one I'm about to create is not one of those documented.

**Describe the bug**
The popout arrow icon on right side of screen that opens sf inspector does not have any alt text/aira label, the action itself does get announced for a screen reader. likely by changing the aria role on the image to be **none** or **presentation** would be the best way to resolve it so it is ignored as the image is purely decorative.
<img width="240" alt="Screenshot 2024-06-15 at 9 10 16 AM" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/103126390/9f6e527c-378d-4b0c-8fbb-fe2f742da437">

**To Reproduce**
1. Go to any org using a browser with SF inspector
2. Turn on chrome inspector
3. Scroll down to observe the arrow image on right side does not have alt text or a role of presentation/none

**Expected behavior**
The role of the arrow icon should be none or presentation
**Screenshots**

Specifically it is the arrow
<img width="594" alt="Screenshot 2024-06-15 at 8 55 50 AM" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/103126390/005a8fe6-a81a-4f27-9527-217a73fd5482">

**Desktop (please complete the following information):**
Happens in all 

**Additional context**
I am not 100% sure but i think it could be fixed like this (for sure do test because like i said, not sure and def not a dev. but would be something like adding this into the code    ` img.setAttribute("role", "presentation"); `

```
  function setRootCSSProperties(rootElement, buttonElement) {
    let popupArrowOrientation = iFrameLocalStorage.popupArrowOrientation ? iFrameLocalStorage.popupArrowOrientation : "vertical";
    let popupArrowPosition = iFrameLocalStorage.popupArrowPosition ? (iFrameLocalStorage.popupArrowPosition + "%") : "122px";
    let img = document.createElement("img");
    img.setAttribute("role", "presentation"); // This is the change that would set the role to presentation
    if (popupArrowOrientation == "vertical") {
      rootElement.style.right = 0;
      rootElement.style.top = popupArrowPosition;
      img.src = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAPCAYAAADd/14OAAAA40lEQVQoz2P4//8/AzpWzGj6L59U/V8urgxMg/g4FUn6J/+X9E38LxWc8V8htR67IpCkuGfMfxCQjSpENRFFkXvk/1+/foGxQloDSD0DVkVfvnyBY7hCdEVv3rxBwXCFIIdKh2WDFT1+/BgDo1qd2fL/1q1bWDFcoW5xz3/Xppn/oycu/X/x4kUMDFeoWdD136R8wn+f9rlgxSdOnEDBKFajK96/fz8coyjEpnj79u1gjKEQXXFE/+L/Gzdu/G9WMfG/am4HZlzDFAf3LPwfOWEJWBPIwwzYUg9MsXXNFDAN4gMAmASShdkS4AcAAAAASUVORK5CYII=";
      buttonElement.classList.add("insext-btn-vertical");
    } else {
      rootElement.style.bottom = "0px";
      rootElement.style.right = popupArrowPosition;
      img.src = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA8AAAAKCAYAAABrGwT5AAAAAXNSR0IArs4c6QAAAFBlWElmTU0AKgAAAAgAAgESAAMAAAABAAEAAIdpAAQAAAABAAAAJgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAD6ADAAQAAAABAAAACgAAAADdC3pnAAABWWlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNi4wLjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgoZXuEHAAABKElEQVQoFWNgwAI0C7r+6xb3/AdJKaTW/1fMaAKz0ZUyoguANHKzszEIcnMy3Hn+muHX2+cMLDwCDExs7Az3Z9ShqGdC1gzTKCHAyyDGz8OwszCM4c/Hdwy/P75l+PfrJwO6C+CakTXyc3EwlDnogM09M6eL4e+Xj1gNAGtG15hrrozsIIarSydjNYARXWOKnhQDJycnBubg4GBQDk5lYObhZ2DlFwaHARMocORFBRl4ONgYYtSEUGxE5zzevJDh77cvwEB8AQ4DJnZWFgY2FmaGSCU+dLVY+S+2LWZg+PeP4f+f3wwsP3//Yfj8/SdD6/G3DK/evceqAVkQFHiMwGhjZGFlYPn68xfDwzfvGX78+sPwYFYDSjwia4KxQdHF/JePgZGZmQEASqV1t0W3n+oAAAAASUVORK5CYII=";
      buttonElement.classList.add("insext-btn-horizontal");
    }
    buttonElement.appendChild(img);
  }

```

**Comment by tprouvot** [comment]
Hi @mavtron-andrewrusso ,
Thanks for pointing this out ! 
Since you already take a look at the code, would you be interested in submitting a PR ?

**Comment by mavtron-andrewrusso** [comment]
Ok, I'll give it a stab.

**Comment by tprouvot** [comment]
@mavtron-andrewrusso it's done 👍 
If you have any other suggestion to improve screen reader accessibility feel free to create new feature request !
